test/role.test.ts

```diff
@@ -312,42 +319,20 @@ Array [
 
     // WHEN
     const role = new kplus.ClusterRole(chart, 'pod-reader');
-    const rule = role.addRule({
-      nonResourceUrls: ['/healthz', '/healthz/*'],
-      verbs: ['get', 'post'],
-    });
+    role.allow(['get', 'post'], kplus.NonApiResource.of('/healthz'), kplus.NonApiResource.of('/healthz/*'));
```

Comment:
Maybe we could allow adding multiple endpoints through a single call? It's a possible future enhancement anyways 🙂 
```suggestion
    role.allow(['get', 'post'], kplus.NonApiResource.of('/healthz', '/healthz/*'));
```